webapp/src/components/createCategory/createCategory.tsx

```diff
@@ -32,35 +32,38 @@ const CreateCategory = (props: Props): JSX.Element => {
             className='CreateCategoryModal'
             onClose={props.onClose}
         >
-            <div className='CreateCategory'>
-                <h3>{props.title}</h3>
-                <input
-                    className='categoryNameInput'
-                    type='text'
-                    placeholder={placeholder}
-                    value={name}
-                    onChange={(e) => setName(e.target.value)}
-                    autoFocus={true}
-                    maxLength={100}
-                />
-                <div className='createCategoryActions'>
-                    <Button
-                        size={'medium'}
-                        danger={true}
-                        onClick={props.onClose}
-                    >
-                        {cancelText}
-                    </Button>
-                    <Button
-                        size={'medium'}
-                        filled={Boolean(name)}
-                        onClick={() => props.onCreate(name)}
-                        disabled={!name}
-                    >
-                        {props.initialValue ? updateText : createText}
-                    </Button>
+            <form>
+                <div className='CreateCategory'>
```

Comment:
We follow the convention of having the component's name as a classname on the outermost HTML element being returned. So to continue following it, you can either set the classname on the `form` or wrap everything inside the `<div className='CreateCategory'>` in a form. Up to you.

Reply:
Thanks for the review @harshilsharma63, sure I will fix it.